content/post/practical_ml.md

```diff
@@ -0,0 +1,77 @@
+
+The Data Scientist may think it is important to know which event came in second or third place. Take, for example, the case when an event is scored as `birthday` with 50% probability and `halloween` with 50% probability. It is likely to be a halloween-themed birthday party. In this case, it is arguable that the best classification is the one which is most useful to the Business. If there is a push to understand seasonal holidays, then adding to the pool of seasonal holidays (i.e. classifying as halloween) could be more valuable than adding to the pool of birthday parties.
+
+It is important to understand what the Business will do with the output of our model. Remember that our ultimate goal is to empower the Business to make the best data-informed decision possible. To that end, providing 3 or 4 best estimates and asking the stakeholder to figure it out is suboptimal. **If they can't make a decision with our output, then our job isn't complete**.
+
+
+## Listen to your surroundings
+
+Ideally you work in an organization with Business stakeholders, Data Engineers and Analysts. Each member plays an important role in the model building and deployment process.
+
+### Business Stakeholders
+
+Stakeholders understand the product, have the context and know what is coming down the pike. Use them. When building our classification algorithm, what features other than text can be used? There may be obvious ones such as `date sent`. But there is a strong chance that product managers know the product much better than we do. Specifically, they will have a good sense for how users engage with the product. In this way, they will be able to qualitatively tell us about differences in users who look for wedding cards and those who look for Mother's Day cards.
+
+Other Business stakeholders will have more context than us. They will know historically which data we should not trust or may have a bias. For example, a specific marketing campaign could have attracted certain types of users. Once that campaign is turned off, there is a good chance those types of users will not be returning. That introduces bias in our model unless we account for it. But gleaning that from a raw database could be difficult.
+
+### Data Analysts
+
+In general, Data Analysts are closer to the Business than Data Scientists. The analyst has a better understanding of the logical data model and should therefore peer review queries. We should also leverage their knowledge of existing queries (e.g. materialized_views) so that we don't duplicate work. 
+
+Worse yet - imagine our query doesn't yield the same expected result as theirs, and it is likely their query that powers Operating reports. Before presenting our conclusions to the Business, we can be sure that they will perform the necessary sniff test to check that our output conforms to their expectations. Avoid the following scenario:
```

Comment:
I'd recommend writing the name of the person (e.g., Data Scientists) instead of some of the pronouns (e.g., theirs). You don't have to do all of them, but could help the reader follow your thought

Reply:
I would also write `Operating` in lower case